lib/dialects/abstract/query-generator.js

```diff
@@ -2245,6 +2248,15 @@ module.exports = (function() {
 
     booleanValue: function(value) {
       return value;
+    },
+
+    geometrySelect: function(column) {
+      // just assuming default behavior will return geo json
+      return this.quoteIdentifiers(column);
+    },
+
+    geometryToGeoJson: function(value) {
+      return JSON.parse(value);
```

Comment:
Not sure parsing belongs in the QG. Perhaps in dialects/X/geojson.js? With a parse and stringify function similar to how PG range works https://github.com/sequelize/sequelize/blob/master/lib/dialects/postgres/range.js